### DISCUSSION TITLE
请问docker安装Moviepilot更新后如何恢复数据的?

### DISCUSSION THREAD

**Discussion by Pt26388** [jxxghp/MoviePilot#2025] [问题咨询]
首先大家是怎么更新版本的?我重新PULL后生成新的容器,环境配置及后台设定,插件里的配置全都要重新设置,太麻烦了.有更好的更新方式吗?
另外插件市场有备份数据,但怎么恢复数据呢?
烦请给点相关指引的链接,谢谢.

**Comment by cng10001** [comment]
不太明白为什么会这样，docker的数据文件夹是不会变的，怎么会要重新设置呢！

**Reply by Pt26388** [reply]
原来是我一直没映射到正确的/config。谢了。
另外，环境变量是另一个文件夹还是要每次都要设置认证？

**Reply by cng10001** [reply]
环境变量不是文件夹，就是一些配置，群晖在编辑容器高级设置里面

**Reply by Pt26388** [reply]
好的，谢谢。
只能每次配一下环境变量了。

**Reply by cng10001** [reply]
![image](https://github.com/jxxghp/MoviePilot/assets/115156545/829f54cf-6f9f-407d-b23a-ac74da9cc501)
其他品牌不清楚，如果你是群晖的话，垃最新版，直接可以重置的，不用删除重新配。而且也可以有重启更新的办法啊

**Reply by Pt26388** [reply]
是群晖，以前一直用套件版，最近更新频繁才用的docker，才知道可以重置更新，谢谢。
